test/polytopes.jl

```diff
@@ -41,7 +41,7 @@
 
     s1 = Segment(P2(1,2), P2(1,0))
     s2 = Segment(P2(1,0), P2(1,1))
-    @test s1 ∩ s2 == Segment(P2(1,1), P2(1,0))
+    @test s1 ∩ s2 == Segment(P2(1,0), P2(1,1))
```

Comment:
That's because the intersection segment is built the other way around?

Reply:
Exactly. In this case the first segment is oriented downwards and the second segment is oriented upwards. The previous implementation was preserved the orientation of the first segment. Now we always return segments oriented towards the positive orthant, which in this case is a segment that points upwards. I think it is a reasonable default, and actually it should be more easy to post-process as a user too. If any algorithm needs a specific orientation convention, it should be easy to post-process the result.